EIPS/eip-4844.md

```diff
@@ -275,7 +275,7 @@ def point_evaluation_precompile(input: Bytes) -> Bytes:
 
 ___WARNING: This is only for testing___
 
-For early draft implementations, we simply change `get_blob_gas(parent)` to always return `SIMPLE_GAS_PER_BLOB`.
+For early draft implementations, we simply change `get_data_gasprice(header)` to always return `SIMPLE_GAS_PER_BLOB`.
```

Comment:
It's probably better to just remove the `Gas price of blobs (Simplified version)` testing-only section. It doesn't translate well to the data-gas approach. The older version was charging a constant amount of regular gas per blob: this is not implemented/used by clients anymore.